sopel_modules/rat_board.py

```diff
@@ -2037,3 +2038,22 @@ def cmd_invalid(bot, trigger, caseid):
 
     except:
         bot.reply('Couldn\'t find a case with id ' + str(caseid) + ' or other APIError')
+
+
+@commands('lang', 'language')
+@parameterize('rw', usage='<case number or name> <language code>')
+@require_permission(Permissions.Rat)
```

Comment:
looking at the `Permissions` enum,we should be using `Permissions.rat` and **not** `Permissions.Rat`. afaik enums are case-sensitive.

Reply:
You're completely right here, that's a typo.